WTWSMS/decisions/WtWSMS_realm_decisions.txt

```diff
@@ -22,8 +22,8 @@ decisions = {
 				narrative_event = { id = guard.3100 }
 				create_character = {
 					random_traits = yes
-					religion = nicene
-					culture = greek
+					religion = ROOT
+					culture = ROOT
```

Comment:
Because the Scholae Palatinae only had Nicene characters historically from what I understand, and was designed in a way that was associated with the Nicene religion. So I would be more inclined to change the culture than religion, if anything.